assets/blocks/RoofTilesSmallTimber.block

```diff
@@ -0,0 +1,10 @@
+/**
+ * Small timber roof tiles
+ */
+{
+    "displayName": "Small Timber Roof Tiles",
+    "shape": "engine:cube",
```

Comment:
I think all the `RoofTiles` blocks should have the same shape, which should either be none (defaults to `engine:cube`)  or something resembling a typical roof, like `engine:slope` or `engine:stair`. Not sure about which, though.... I tend towards not specifying a shape at all.

Reply:
One thing I noticed while updating the template is that the explicit `engine:cube` shape will (a) prevent the blocks from being available in other shapes, like slopes or stairs, and (b) even seem to break the template when you want to use them as plain cubes...

Taking all `"shape": "engine:cube"` out should help.

Reply:
In structure templates, different shapes and orientations can be specified with
```
"blockType": "StructuralResources:RoofTilesSmallTimber:engine:SlopeCorner.BACK
```
as described in https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/world/block/BlockUri.java#L31